tests/validation/tests/v3_api/test_airgap.py

```diff
@@ -0,0 +1,384 @@
+import os
+import pytest
+import time
+from .test_custom_host_reg import (
+    random_test_name, RANCHER_SERVER_VERSION, HOST_NAME, AGENT_REG_CMD
+)
+from .common import (
+    AmazonWebServices, run_command, TEST_IMAGE, TEST_IMAGE_NGINX,
+    TEST_IMAGE_OS_BASE
+)
+
+PRIVATE_REGISTRY_USERNAME = os.environ.get("RANCHER_BASTION_USERNAME")
+PRIVATE_REGISTRY_PASSWORD = os.environ.get("RANCHER_BASTION_PASSWORD")
+PROVIDER_ID = os.environ.get("RANCHER_PROVIDER_ID", "")
```

Comment:
In the case where I want to get both the bastion server and a rancher server in one job, should this value be set by the job itself?

Reply:
`PRIVATE_REGISTRY_USERNAME` and `PRIVATE_REGISTRY_PASSWORD` are set by the job, and `PROVIDER_ID` isn't necessary in this case as the test to create both bastion server and rancher already knows about the bastion server.

Reply:
@rancher-max  thanks for the explanation. 

Why it is called `PROVIDER_ID`? Could we make the name more self-explaining?